docs/_docs/posts.md

```diff
@@ -207,18 +207,19 @@ filter by tags or any other variable created with extensions.
 ## Post excerpts
 
 Each post automatically takes the first block of text, from the beginning of
-the content to the first occurrence of `excerpt_separator`, and sets it as the `post.excerpt`.
+the content to the first occurrence of `excerpt_separator`, and sets it in the
+post's data hash.
 Take the above example of an index of posts. Perhaps you want to include
 a little hint about the post's content by adding the first paragraph of each of
 your posts:
 
 {% raw %}
 ```liquid
 <ul>
-  {% for post in site.posts %}
+  {% for doc in site.posts %}
```

Comment:
Why change this? Makes it harder to understand for new users, IMO.

Reply:
.. so that they do not think that one should always just use `post` parameter for posts..

Reply:
But it does make more sense semantically to say `for post in posts`, doesn't it? I think this example is fine, as long as we clarify it explicitly somewhere else.

Reply:
okay.. I'll undo this change.